app/controllers/api/v1/sessions_controller.rb

```diff
@@ -5,7 +5,7 @@ def create
     if user && user.valid_password?(params[:password]) 
       User.current_user = user
       env['warden'].set_user(user)
-      render json: user.as_json(only: [:id, :admin, :email, :confirmed_at])
+      render json: user.as_json(only: [:id, :admin, :superuser, :email, :confirmed_at])
```

Comment:
I believe all these `superuser`s were recently renamed to `superadmin`.

Reply:
done